resources/assets/components/utilities/ScholarshipCard/ScholarshipCard.js

```diff
@@ -49,6 +55,19 @@ const ScholarshipCard = ({ campaign }) => {
     { height: 820, width: 1460 },
   ]);
 
+  const handleScholarshipCardShareClick = () => {
+    trackAnalyticsEvent('clicked_scholarship_gallery_block_apply_now', {
+      action: 'button_clicked',
+      category: EVENT_CATEGORIES.siteAction,
+      label: 'scholarship_gallery_card',
+      context: {
+        url: path,
+        campaignId,
+        pageId: getPageContext(),
```

Comment:
I don't think we can full on use the `getPageContext` helper here since it returns an object with `campaignId` & `pageId` (and it's pulling the `campaignId` from the `window.STATE.campaign` which wouldn't be populated on most other pages). 

But hmm what we can actually do here is use the spread operator (`...`) to add both fields returned from `getPageContext` directly as fields on the `context` object, and then overwrite the `campaignId` below it:
```suggestion
        ...getPageContext(),
        campaignId,
```

Reply:
I love this suggestion! Done. 
I tested it and it did return an object